src/cmd/domain.go

```diff
@@ -0,0 +1,233 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"github.com/creasty/defaults"
+	"github.com/opslevel/cli/common"
+	"github.com/opslevel/opslevel-go/v2023"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"strings"
+)
+
+var assignSystemCmd = &cobra.Command{
+	Use:        "system ID|ALIAS SYSTEM",
+	Short:      "Add a system to a domain",
+	Example:    `opslevel create system my_domain my_system`,
+	Args:       cobra.ExactArgs(2),
+	ArgAliases: []string{"DOMAIN_ID", "DOMAIN_ALIAS", "SYSTEM"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		system := args[1]
+
+		domain, err := getClientGQL().GetDomain(key)
+		cobra.CheckErr(err)
+		common.WasFound(domain.Id == "", key)
+
+		addErr := domain.AssignSystem(getClientGQL(), system)
+		cobra.CheckErr(addErr)
+		fmt.Printf("add system '%s' to domain '%s'\n", system, domain.Name)
+	},
+}
+
+var createDomainCmd = &cobra.Command{
+	Use:   "domain",
+	Short: "Create a domain",
+	Long: `Create a domain
```

Comment:
cobra has a new field `Example` that you can use for examples.  Checkout the user.go for an example